Gemfile

```diff
@@ -63,8 +72,6 @@ group :development do
   gem 'better_errors'
   gem 'binding_of_caller'
   gem 'bullet'
-  gem 'devise'
-  gem 'faker'
   gem 'pry-rails'
   gem 'rack-canonical-host'
   gem 'rails_db'
```

Comment:
I did explore the `rails-erd` gem noticing that you had added this in the Gemfile and talked about it in your video. This was really awesome, since ERD models tend to get very complicated over time and this gives a quick way to re-create the ERD diagram.